publish-imgs

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# vim: ft=sh sw=2 et
+# shellcheck shell=bash
+set -efo pipefail
+
+test -z $1 && (echo "pass in img" && exit 1)
+
+IMG_DIR="imgs/$1"
+
+# Make sure nix-build was run
+test -f ${IMG_DIR}/result || (echo "please build derivation" && exit 1)
+
+derivation=$(nix show-derivation $(nix-instantiate ${IMG_DIR}/default.nix))
```

Comment:
What if someone doesn't name their derivation `default.nix`? For example: M2's MR for Yubikey uses `imgs/yubikey/nixos_yubikey.nix`. Should we include in the top level `README` to always name derivations `default.nix`?

Reply:
`default.nix` is a nix standard convention but your point is still valid that we could have other derivations that we would need to support. @Lynty would you mind just opening up an issue in this repo and well circle back on supporting any `.nix` file declared for a docker image? That seems pretty easy to pass it to this script but not needed at the moment.

In the case of #3, this image is not a docker image but an iso image. So its actually something we should probably move out of the `img/` and into somewhere else so its not confusing.